main.go

```diff
@@ -32,6 +32,8 @@ var (
 	useCIE2000 bool
 	// Threshold amount for Euclidean method.
 	useThreshold int
+	// Transparent colour value.
+	nColorKey int
```

Comment:
Hungarian notation is not used so much in Go. The transition from C also took me some time to get familiar with new naming conventions. Using `colorKey` as variable name should work well enough.

Reply:
I understand. Should I go ahead and change it to `colourKey`? :P